src/openrct2-ui/windows/EditorBottomToolbar.cpp

```diff
@@ -310,7 +310,8 @@ static void window_editor_bottom_toolbar_mouseup([[maybe_unused]] rct_window* w,
     if (widgetIndex == WIDX_PREVIOUS_STEP_BUTTON)
     {
         if ((gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER)
-            || (gSpriteListCount[SPRITE_LIST_FREE] == MAX_SPRITES && !(gParkFlags & PARK_FLAGS_SPRITES_INITIALISED)))
+            || (gSpriteListCount[static_cast<uint8_t>(EntityListId::Free)] == MAX_SPRITES
```

Comment:
This is a bit unwieldy. Is there no way around this casting requirement?

(If there isn't, I prefer saving `(gSpriteListCount[static_cast<uint8_t>(EntityListId::Free)]` into an intermediate variable.)

Reply:
There is no way to have it automatically convert into a uint8_t unfortunately but we could make an assessor function that gets the list count from a entity list id.

Reply:
That would also work. I'll leave it to you which one you pick.

Reply:
I have applied the accessor function change. Eventually I'm going to remove the setters completely so I haven't created a Set function.